mexican-standoff.json

```diff
@@ -0,0 +1,287 @@
+{
+  "player_count": 3,
+  "labeling": [
+    [],
+    [
+      "alive3"
+    ],
+    [
+      "alive2"
+    ],
+    [
+      "alive2",
+      "alive3"
+    ],
+    [
+      "alive1"
+    ],
+    [
+      "alive1",
+      "alive3"
+    ],
+    [
+      "alive1",
+      "alive2"
+    ],
+    [
+      "alive1",
+      "alive2",
+      "alive3"
+    ]
+  ],
+  "transitions": {
```

Comment:
This is incorrectly formatted. It should be using arrays instead of objects

Reply:
I believe dicts improve readability a lot. E.g. the transition:

> from state 0, p1 choose 0, p2 choose 0, p3 choose 0, results in 0

is expressed in dicts as: `0: {0: {0: {0: {0}}}}`
and in lists: `[[[[[0]]]]` where all other transitions are omitted.

Reply:
And for reading in the json in the model-checker, both can work.

Reply:
I would strongly prefer having it as list right now. We can look into having objects at a later point in time. Using lists simplifies the deserialization

Reply:
I've added support for both in efc324c